core/templates/expressions/expression-syntax-tree.service.ts

```diff
@@ -201,120 +229,120 @@ export class ExpressionSyntaxTreeService {
     '+': {
       eval: (args: Array<string>): number => {
         this.verifyNumArgs(args, 1, 2);
-        var numericArgs = this._coerceAllArgsToNumber(args);
+        var numericArgs = this.coerceAllArgsToNumber(args);
         return numericArgs.length === 1 ?
           numericArgs[0] :
           numericArgs[0] + numericArgs[1];
       },
       getType: (args: Array<string>): string => {
         this.verifyNumArgs(args, 1, 2);
-        this._verifyArgTypesMatchExpectedType(args,
+        this.verifyArgTypesMatchExpectedType(args,
           AppConstants.PARAMETER_TYPES.REAL);
         return AppConstants.PARAMETER_TYPES.REAL;
       }
     },
     '-': {
       eval: (args: Array<string>): number => {
         this.verifyNumArgs(args, 1, 2);
-        var numericArgs = this._coerceAllArgsToNumber(args);
+        var numericArgs = this.coerceAllArgsToNumber(args);
         return numericArgs.length === 1 ?
           -numericArgs[0] :
           numericArgs[0] - numericArgs[1];
       },
       getType: (args: Array<string>): string => {
         this.verifyNumArgs(args, 1, 2);
-        this._verifyArgTypesMatchExpectedType(args,
+        this.verifyArgTypesMatchExpectedType(args,
           AppConstants.PARAMETER_TYPES.REAL);
         return AppConstants.PARAMETER_TYPES.REAL;
       }
     },
     '*': {
       eval: (args: Array<string>): number => {
         this.verifyNumArgs(args, 2);
-        var numericArgs = this._coerceAllArgsToNumber(args);
+        var numericArgs = this.coerceAllArgsToNumber(args);
         return numericArgs[0] * numericArgs[1];
       },
       getType: (args: Array<string>): string => {
         this.verifyNumArgs(args, 2);
-        this._verifyArgTypesMatchExpectedType(args,
+        this.verifyArgTypesMatchExpectedType(args,
           AppConstants.PARAMETER_TYPES.REAL);
         return AppConstants.PARAMETER_TYPES.REAL;
       }
     },
     '/': {
       eval: (args: Array<string>): number => {
         this.verifyNumArgs(args, 2);
-        var numericArgs = this._coerceAllArgsToNumber(args);
+        var numericArgs = this.coerceAllArgsToNumber(args);
         return numericArgs[0] / numericArgs[1];
```

Comment:
Should we be checking for division by zero here? @seanlip

Reply:
No need for now -- let's keep parity with the existing code. Thanks!